### DISCUSSION TITLE
Questions about continuant fiat boundaries

### DISCUSSION THREAD

**Discussion by gregfowlerphd** [BFO-ontology/BFO-2020#106] [General]
This isn't really a bug, just a series of questions (though I suppose it's possible they might lead to a bug being uncovered).

The elucidation of ‘continuant fiat boundary’ reads:

> A continuant fiat boundary b is an immaterial entity that is of zero, one or two dimensions & such that there is no time t when b has a spatial region as continuant part & whose location is determined in relation to some material entity

The final clause here suggests, without outright stating, that every CFB is a _boundary of_ some material entity. Is that suggestion correct?

Also, while the elucidation indicates that every CFB’s location is determined in relation to some material entity, can a CFB’s location _also_ be determined in relation to something that _isn’t_ a material entity? And can a CFB be a _boundary of_ something that isn’t a material entity?

**Comment by wceusters** [comment]
This part of BFO is indeed not well documented and worked out. Michael and I looked a bit deeper in issues and possibilities re 'surface' a subtype of CFB. See https://www.utwente.nl/en/eemcs/fois2024/resources/papers/ceusters-fiat-surfaces-in-basic-formal-ontology.pdf and in that document a link to several possible more elaborated views.

**Reply by gregfowlerphd** [reply]
@wceusters: Thanks for the link, Werner. Really look forward to reading the paper!

**Comment by johnbeve** [comment]
@gregfowlerphd @wceusters 
Moving this back to the Issue tracker; apologies for the back and forth. See issue here https://github.com/BFO-ontology/BFO-2020/issues/107
